src/Xamarin.Android.Build.Tasks/Tasks/Aot.cs

```diff
@@ -203,39 +203,47 @@ static int GetNdkApiLevel(string androidNdkPath, string androidApiLevel, Android
 		}
 
 		bool DoExecute () {
-			Log.LogDebugMessage ("Aot:", AndroidAotMode);
-			Log.LogDebugMessage ("  AndroidApiLevel: {0}", AndroidApiLevel);
-			Log.LogDebugMessage ("  AndroidAotMode: {0}", AndroidAotMode);
-			Log.LogDebugMessage ("  AndroidSequencePointsMode: {0}", AndroidSequencePointsMode);
-			Log.LogDebugMessage ("  AndroidNdkDirectory: {0}", AndroidNdkDirectory);
-			Log.LogDebugMessage ("  AotOutputDirectory: {0}", AotOutputDirectory);
-			Log.LogDebugMessage ("  EnableLLVM: {0}", EnableLLVM);
-			Log.LogDebugMessage ("  IntermediateAssemblyDir: {0}", IntermediateAssemblyDir);
-			Log.LogDebugMessage ("  LinkMode: {0}", LinkMode);
-			Log.LogDebugMessage ("  SdkBinDirectory: {0}", SdkBinDirectory);
-			Log.LogDebugMessage ("  SupportedAbis: {0}", SupportedAbis);
-			Log.LogDebugTaskItems ("  ResolvedAssemblies:", ResolvedAssemblies);
-			Log.LogDebugTaskItems ("  AdditionalNativeLibraryReferences:", AdditionalNativeLibraryReferences);
+			LogDebugMessage ("Aot:", AndroidAotMode);
+			LogDebugMessage ("  AndroidApiLevel: {0}", AndroidApiLevel);
```

Comment:
For future changes -- not this PR -- we should use C#6 string interpolation and `nameof()` to assist in refactoring:

    LogDebugMessage ($"  {nameof (AndroidApiLevel)}: {AndroidApiLevel}");